quarkus-modules/quarkus-testcontainers/customer-service/src/test/java/info/customer/CustomerResourceTest.java

```diff
@@ -0,0 +1,48 @@
+package info.customer;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.common.http.TestHTTPEndpoint;
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+@QuarkusTestResource(TestcontainersManager.class)
+@QuarkusTest
+@TestHTTPEndpoint(CustomerResource.class)
+class CustomerResourceTest {
```

Comment:
We have a convention to name heavy tests like this *LiveTest, which will exclude running it from the default CI build:
```suggestion
class CustomerResourceLiveTest {
```